update_test.go

```diff
@@ -6,6 +6,7 @@ import (
 )
 
 func TestDynamicUpdateParsing(t *testing.T) {
+	// This does nothing with dynamic updates, move elsewhere?
```

Comment:
It does actually, rr parsing with any rdata should work (for all types), i.e.
`miek.nl. IN A` is vaild.